dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -69,12 +71,50 @@ const utilisationReportsRouter = express.Router();
  */
 utilisationReportsRouter.route('/').post(postUploadUtilisationReportPayloadValidator, postUploadUtilisationReport);
 
+/**
+ * @openapi
+ * /utilisation-reports/validate:
```

Comment:
Don't feel too strongly about this, but `validate` feels like it may be too broad and it might be better to use something more specific like `validate-report-csv`. What do you think?

Reply:
I've gone for `report-data-validation`, let me know if that is ok!

Reply:
Sounds good to me